tests/test_2_neurolucida.py

```diff
@@ -409,13 +356,7 @@ def test_single_children():
                       )
                  ''') as tmp_file:
 
-        with captured_output() as (_, err):
```

Comment:
I cannot comment on it but shouldn't you remove the ; merged with parent section from the tmp_asc_file ?

Reply:
Ah yes good point.